recipes/libblitz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.10" %}
+
+package:
+  name: libblitz
+  version: {{ version }}
+
+source:
+  fn: blitz-{{ version }}.tar.gz
+  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
+  md5: 66268b92bda923735f2e3afc87dcb58a
+
+build:
+  number: 0
+  skip: true   # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+
+test:
+  commands:
+    - test -d $PREFIX/include/blitz  # [unix]
+
+about:
+  home: https://sourceforge.net/projects/blitz
+  license: Artistic License 2.0, BSD, LGPLv3
```

Comment:
The projects page is super confusing on this point. According to their [webpage]( http://blitz.sourceforge.net/ ), its `LGPL 3`. Could we change this to match?